apps/aecore/lib/aecore/structures/account_handler.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Aecore.Structures.AccountHandler do
```

Comment:
can you rename this with semantic meaning? `Handler` doesn't say what its role is

Reply:
@thepiwo  I think we can move all the functionality to account.ex module. What do you think about it?

Reply:
If this wouldn't make account to to big, yes, otherwise find some better name, as abstraction and module separation are preferred.